yamlgen/deploy/brupop-resources.yaml

```diff
@@ -93,7 +101,7 @@ metadata:
   name: brupop-apiserver
   namespace: brupop-bottlerocket-aws
 spec:
-  replicas: 3
+  replicas: 1
```

Comment:
Why we do change the number of API server from 3 to 1. I haven't finished reviewing, so  might be able to find the answer somewhere : )

Reply:
The spec in the code was still 3 though, do you manually changed to 1 for testing purpose?

```
        spec: Some(DeploymentSpec {
            replicas: Some(3),
```

Reply:
Yeah, my mistake! I've fixed this in the latest revision. I just wanted to make it easier to grab all of the logs.